Atomic/help.py

```diff
@@ -77,29 +76,26 @@ def man_help(self, docker_id):
         """
         if not os.path.exists(self.mount_location):
             os.makedirs(self.mount_location)
-        # Set the pager to less -R
-        enc = sys.getdefaultencoding()
         dm = mount.DockerMount(self.mount_location, mnt_mkdir=True)
-        mnt_path = dm.mount(docker_id)
-        help_path = os.path.join(mnt_path, self.help_file_name)
-        if not os.path.exists(help_path):
-            help_path = os.path.join(mnt_path, 'rootfs', self.help_file_name)
-        try:
-            help_file=open(help_path)
-        except IOError:
-            dm.unmount(path=mnt_path)
-            raise ValueError("Unable to find help file for {}".format(self.docker_object))
+        with mount.MountContextManager(dm, docker_id) as dmcm:
+            help_path = os.path.join(dmcm.mnt_path, self.help_file_name)
+            if not os.path.exists(help_path):
+                help_path = os.path.join(dmcm.mnt_path, 'rootfs', self.help_file_name)
+            try:
+                help_file=open(help_path)
+            except IOError:
+                raise ValueError("Unable to find help file for {}".format(self.docker_object))
 
-        cmd2 = ['/usr/bin/groff', '-man', '-Tascii']
-        if not os.path.exists(cmd2[0]):
-            raise IOError("Cannot display help file for {}: groff unavailable".format(self.docker_object))
+            groff_cmd = '/usr/bin/groff'
+            cmd = [groff_cmd, '-man', '-Tascii']
+            if not os.path.exists(groff_cmd):
+                raise IOError("Cannot display help file for {}: {} unavailable.".format(
+                    self.docker_object, groff_cmd))
```

Comment:
FWIW, I think it would be better to have some kind of indication that the groff cmd was not found in the error message rather than the generic "can't display help file for".  Perhaps:

raise IOError("Cannot display help file for {} as the groff command was not found at: {} unavailable.".format(